<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Beautifier::Internal::ClangFormat::ClangFormatOptionsPage</class>
 <widget class="QWidget" name="Beautifier::Internal::ClangFormat::ClangFormatOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="options">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QRadioButton" name="useCustomizedStyle">
        <property name="text">
         <string>Use customized style:</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="Beautifier::Internal::ConfigurationPanel" name="configurations" native="true"/>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="usePredefinedStyle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Use predefined style:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="predefinedStyle"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="fallbackStyleLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Fallback style:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="fallbackStyle">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="configuration">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="commandLabel">
        <property name="text">
         <string>Clang Format command:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Utils::PathChooser" name="command" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mimeLabel">
        <property name="text">
         <string>Restrict to MIME types:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mime"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Beautifier::Internal::ConfigurationPanel</class>
   <extends>QWidget</extends>
   <header location="global">beautifier/configurationpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
